sdk/PrebidServerAdapter/PBServerAdapter.m

```diff
@@ -118,6 +119,9 @@ - (NSDictionary *)requestBodyForAdUnits:(NSArray<PBAdUnit *> *)adUnits {
     if (keywords) {
         requestDict[@"keywords"] = keywords;
     }
+#ifdef DEBUG
+	requestDict[@"is_debug"] = @(YES);
```

Comment:
Does this mean when the framework build type is debug, then add is_debug to the request?

Reply:
Yes